tutorials/rdf_fitting_demo.ipynb

```diff
@@ -22,11 +22,11 @@
     "import numpy as np\n",
```

Comment:
<p>Do we want to keep this example for now but change it to something else down the line now that the <code>scipy</code> method is actually in the library?  I will raise an issue. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/solvation-analysis/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Though we now use scipy under the hood, it's still an excellent tutorial on implementing your own rdf kernel. 

The internal implementation is slightly different too, so I don't see a big need to change it.